src/key.js

```diff
@@ -26,7 +26,12 @@ const pathSep = pathSepB[0]
  *
  */
 class Key {
+  /**
+   * @param {string | Uint8Array} s
+   * @param {boolean} [clean]
+   */
   constructor (s, clean) {
+    Object.defineProperty(this, symbol, { value: true })
```

Comment:
I would much rather do the getter instead (as I did in bunch of other places where I removed is-class)

```suggestion
```

I'll add a suggestion for it down below